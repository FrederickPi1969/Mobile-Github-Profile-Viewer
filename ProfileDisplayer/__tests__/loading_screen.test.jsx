import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import LoadingScreen from '../src/loading_screen';

/**
 * Test the rendering of loading screen.
 * Notice all of the screens that involve fetching
  * (Home, Profile, Repo, Follower, Following)
  * will invoke this loading screen so that we
  * don't need to test repetitively.
  */
it('renders correctly', () => {
  const tree = renderer.create(<LoadingScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
