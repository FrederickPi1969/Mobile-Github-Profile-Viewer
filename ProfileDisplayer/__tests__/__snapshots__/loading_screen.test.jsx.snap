// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "fontSize": 20,
        "fontWeight": "bold",
        "marginTop": "35vh",
        "textAlign": "center",
      }
    }
  >
    Loading...
  </Text>
</View>
`;
